Restaurant.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import RestaurantModel from './RestaurantModel'
+
+class Restaurant extends React.Component {
+  constructor(props) {
+    super(props)
+    this.displayName = "Restaurant"
+  }
+
+  render() {
+    // helper for code simplicity.
+    var model = this.props.model
```

Comment:
My direction is that `ResaurantModel` is a JSON coming from REST end point, I'd assume it won't be a React object. Open for discussion..